source/layouts/partials/footer.html

```diff
@@ -13,6 +13,9 @@
         </ul>
     </nav>
     <div class="socials">
+        <a href="{{ .Site.Params.Socials.Mastodon }}" rel="me" class="mastodon" title="Mastodon">
```

Comment:
This `rel="me"` should do the job of giving us a lovely green tick on our Mastodon account to 'verify' it.